main.gd

```diff
@@ -225,7 +240,7 @@ func _compose_url(urlpath, parameters={}):
 func _on_HTTPRequest_request_completed(result, response_code, headers, body):
 	busy = false
 	if result != OK:
-		emit_signal('gamejolt_request_completed')
+		emit_signal('gamejolt_request_completed',lasttype,"error")
```

Comment:
I think it should be organized in another way - it should put as response an Direction with specific fields, which are saying about response, failure, errors and so on.